resources/views/user/printing.blade.php

```diff
@@ -11,14 +11,14 @@
                     @foreach($user->freePages->sortBy('deadline') as $page)
                         <li>
                         <span class="new badge
-                            @if($page->available())
+                            @if($page->available)
```

Comment:
`available` is a function, not an attribute. It should be `available()`

Reply:
It is an attribute though.